Tests/GRPCCoreTests/RPCErrorTests.swift

```diff
@@ -14,114 +14,144 @@
  * limitations under the License.
  */
 import GRPCCore
-import XCTest
-
-final class RPCErrorTests: XCTestCase {
-  private static let statusCodeRawValue: [(RPCError.Code, Int)] = [
-    (.cancelled, 1),
-    (.unknown, 2),
-    (.invalidArgument, 3),
-    (.deadlineExceeded, 4),
-    (.notFound, 5),
-    (.alreadyExists, 6),
-    (.permissionDenied, 7),
-    (.resourceExhausted, 8),
-    (.failedPrecondition, 9),
-    (.aborted, 10),
-    (.outOfRange, 11),
-    (.unimplemented, 12),
-    (.internalError, 13),
-    (.unavailable, 14),
-    (.dataLoss, 15),
-    (.unauthenticated, 16),
-  ]
+import Testing
 
+@Suite("RPCError Tests")
+struct RPCErrorTests {
+  @Test("Custom String Convertible")
   func testCustomStringConvertible() {
-    XCTAssertDescription(RPCError(code: .dataLoss, message: ""), #"dataLoss: """#)
-    XCTAssertDescription(RPCError(code: .unknown, message: "message"), #"unknown: "message""#)
-    XCTAssertDescription(RPCError(code: .aborted, message: "message"), #"aborted: "message""#)
+    #expect(String(describing: RPCError(code: .dataLoss, message: "")) == #"dataLoss: """#)
+    #expect(String(describing: RPCError(code: .unknown, message: "message")) == #"unknown: "message""#)
+    #expect(String(describing: RPCError(code: .aborted, message: "message")) == #"aborted: "message""#)
 
     struct TestError: Error {}
-    XCTAssertDescription(
-      RPCError(code: .aborted, message: "message", cause: TestError()),
+    #expect(
+      String(describing: RPCError(code: .aborted, message: "message", cause: TestError()))
+      ==
       #"aborted: "message" (cause: "TestError()")"#
     )
   }
 
+  @Test("Error from Status")
   func testErrorFromStatus() throws {
     var status = Status(code: .ok, message: "")
     // ok isn't an error
-    XCTAssertNil(RPCError(status: status))
+    #expect(RPCError(status: status) == nil)
 
     status.code = .invalidArgument
-    var error = try XCTUnwrap(RPCError(status: status))
-    XCTAssertEqual(error.code, .invalidArgument)
-    XCTAssertEqual(error.message, "")
-    XCTAssertEqual(error.metadata, [:])
+    var error = RPCError(status: status)
+    try #require(error != nil)
```

Comment:
Is it possible to do `var error = try #require(RPCError(status: status))` so we can avoid the `!` below?

Reply:
Oh yeah! I swear I thought I'd tried this - will fix